tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -236,7 +236,7 @@ public static Result IsAzureServiceBusInboundV0(this MockSpan span) => Result.Fr
 
         public static Result IsAzureServiceBusOutboundV0(this MockSpan span) => Result.FromSpan(span)
             .Properties(s => s
-                .Matches(Name, "Message")
+                .Matches(Name, "producer")
```

Comment:
Would be interested in what Zach says about this one, because IIRC the way the `Message` span is used is... interesting...

Reply:
For these particular ones, the ResourceName is still `"Message"` so I think this is fine. My one suggestion here is to add a `Matches` clause to assert that the name is always `"Message"`, but unless you have other suggestions it's not worth the CI cycles and re-approval loop 😅

Reply:
Interestingly, the latest messaging conventions (because they're always a-changing) includes a new operation name for `create` which would correspond to these spans: https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#operation-names

If the Azure SDK updates to setting this as the `messaging.operation` attribute, then our operation name logic will (correctly) update the operation name to `servicebus.create`

I do wonder if we should get ahead of that and just add the `messaging.operation` attribute ourselves 🤔 Thoughts?

Reply:
I think we could have this discussion and potentially follow-up work in a separate PR